test/module/irohad/execution/command_validate_execute_test.cpp

```diff
@@ -505,13 +493,11 @@ TEST_F(SubtractAssetQuantityTest, InvalidWhenWrongPrecision) {
  */
 TEST_F(SubtractAssetQuantityTest, InvalidWhenNoAccount) {
   // TODO 2018-04-20 Alexey Chernyshov - IR-1276 - rework with CommandBuilder
-  command = clone(*(TestTransactionBuilder()
-                        .subtractAssetQuantity("noacc", kAssetId, "12.04")
-                        .build()
-                        .commands()
-                        .front()));
+  command = buildCommand(TestTransactionBuilder().subtractAssetQuantity(
+      "noacc", kAssetId, "12.04"));
```

Comment:
Seems like the value is overflowed, so it may cause false positive:
https://github.com/hyperledger/iroha/blob/c4a722d16e4f2bee744eadd623f748175741f3d0/test/module/irohad/execution/command_validate_execute_test.cpp#L435-L438

Looks below also